src/keria/app/agenting.py

```diff
@@ -308,15 +312,15 @@ def __init__(self, hby, rgy, agentHab, agency, caid, **opts):
                                      vry=self.verifier)
 
         doers.extend([
-            Initer(agentHab=agentHab, caid=caid),
+            Initer(agentHab=agentHab, caid=caid, metrics = self.agency.metrics),
```

Comment:
This is the list of doers, you have to include `self.interceptor` in this list to get the recur method to be called as a background process.

Reply:
Can I not put it when I Initialize the agency `super(Agency, self).__init__(doers=[self.interceptor], always=True)` on Line 132 instead of having a logger per agent, I was thinking that since we have a queue per agency instead of agent

Reply:
@pfeairheller I also tried passing `self.agency.interceptor` but it still not running. Does Interceptor need to be Just a Doer Since Agent is a DoDoer?